lib/compress.js

```diff
@@ -1273,7 +1273,21 @@ merge(Compressor.prototype, {
         def(AST_Constant, return_false);
         def(AST_This, return_false);
 
+        var pure_regex = /[@#]pure/;
```

Comment:
The devil in me thinks `https://docs.read.me/notes#pure_in_title` could trip on this.